src/routes/TraderForm/Main/PoolHeader/index.tsx

```diff
@@ -24,20 +33,22 @@ export const PoolHeader: React.FunctionComponent<PoolHeaderProps> = () => {
             labelTypographyToken="primaryBodyXSmallRegular"
             token="%"
             typographyToken="secondaryBodyMediumBold"
-            value={5.33}
+            value={fixedRateInfo.status !== 'success' ? '--' : formatNumber(fixedRateInfo.value)}
           />
         </FixedBox>
         <VariableBox>
           <LabelTokenTypography
             colorToken="lavenderWeb"
-            differenceValue={-2}
+            // differenceValue={-2} //TODO Alex
```

Comment:
forgotten TODO or left for future?

Reply:
Left for future, as this data must be taken from historical data form Ioana

Reply:
[957310c](https://github.com/Voltz-Protocol/voltz-ui/pull/571/commits/957310cc8a0b03d2680b71d8f8ec5e98803f7150)